app/models/manageiq/providers/hawkular/middleware_manager/refresh_parser.rb

```diff
@@ -138,23 +187,62 @@ def process_availability(availability)
       end
 
       def parse_deployment(server, deployment)
-        {
+        specific = {
           :name              => parse_deployment_name(deployment.id),
-          :middleware_server => server, # TODO: does that make sense? What is better?
-          :nativeid          => deployment.id,
-          :ems_ref           => deployment.path
+          :middleware_server => server,
         }
+        parse_base_item(deployment).merge(specific)
```

Comment:
it's the very last statement that will be returned, so why using the mutable variant? I am trying to avoid those mutable `foo!` whenever possible ;)